site/src/app/v2/trouver-un-club/components/club-filters/geolocation-filter/GeolocationFilter.tsx

```diff
@@ -0,0 +1,61 @@
+'use client';
+
+import Range from '@codegouvfr/react-dsfr/Range';
+import styles from './styles.module.scss';
+import cn from 'classnames';
+import { useContext, useEffect, useState } from 'react';
+import { GeolocationContext } from '@/store/geolocationContext';
+import { useSearchParams } from 'next/navigation';
+import { SEARCH_QUERY_PARAMS } from '@/app/constants/search-query-params';
+import { DEFAULT_DISTANCE } from 'utils/map';
+
+interface Props {
+  onChanged: (distance: string) => void;
+}
+
+const GeolocalisationFilter: React.FC<Props> = ({ onChanged }) => {
+  const { loading, error } = useContext(GeolocationContext);
+
+  const searchParam = useSearchParams();
+  const defaultDistance =
+    (searchParam && searchParam.get(SEARCH_QUERY_PARAMS.distance)) || DEFAULT_DISTANCE.toString();
+
+  const [selectedDistance, setSelectedDistance] = useState<string | null>(null);
+
+  useEffect(() => {
+    const identifier = setTimeout(() => {
+      if (selectedDistance) {
+        onChanged(selectedDistance);
+        setSelectedDistance(null);
+      }
+    }, 300);
```

Comment:
Je suggère de mettre 700.
300 c'est trop court, l'event matomo se trigger inutilement lorsqu'on bouge le slider un peu trop lentement
```suggestion
    }, 700);
```

Reply:
J'avais mis 300 pour garder de la fluidité en évitant d'attendre trop longtemps avant la pris en compte du chgmt de distance. Je mets 700 et on verra à l'usage.